recipes/mutpy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mutpy" %}
+{% set version = "0.6.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/MutPy-{{ version }}.tar.gz
+  sha256: 5a13ff28f30fb326cb6d9e9afe7431f36c156c00ff12ebf46f5fae1f7546bb3c
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.8
```

Comment:
Do we know that this doesn't work for python >3.7?

Reply:
Yes, because the package I am trying to add is a fork of this that makes it work for 3.8